spec/feature/user_share_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'user/SharesController', type: :feature do
+  describe '#new' do
+    context 'A User with an associated Birth Records' do
+      let(:user) { FactoryBot.create(:user) }
+      let(:birth_records) { FactoryBot.create_list(:birth_record, 3) }
+      let(:target_birth_record) { FactoryBot.create(:birth_record) }
+
+      before do
+        login_as(user, scope: :user)
+        user.birth_records << birth_records
+        user.birth_records << target_birth_record
+      end
+
+      context 'the user chooses to share the birth record' do
+        before do
+          visit user_birth_records_path
+          find(".user-birth-record[data-id='#{target_birth_record.id}']").click_link('share')
+
+          # wait for destination page to load
+          page.has_css?('body.user__shares--new')
```

Comment:
```suggestion
          page.has_css?('body.user__shares--new')
          Percy.snapshot(page, name: 'share the birth record')
```

Reply:
The `before` block will run twice which probably isn't what you expected. I've added the percy snapshot to line 36 below